rel-eng/tito.props

```diff
@@ -133,6 +133,7 @@ whitelist = foreman
   rubygem-gridster-rails
   rubygem-hammer_cli
   rubygem-hammer_cli_foreman
+  rubygem-foreman_maintain
```

Comment:
Based on @iNecas response, this should be moved up a section to `foreman-nightly-nonscl-rhel7`